specs/UrbanMobility/GtfsRoute/schema.json

```diff
@@ -41,6 +41,9 @@
         "operatedBy": {
           "type": "string",
           "format": "uri"
+        },
+        "shortName": {
+          "type": "string"
```

Comment:
isn't this required?

Reply:
The specification [here](https://developers.google.com/transit/gtfs/reference/#routestxt) states:
> At least one of route_short_name or route_long_name must be specified, or potentially both if appropriate. If the route does not have a short name, please specify a route_long_name and use an empty string as the value for this field.

So I guess the fix should be (following the approach already used for POIs in this [PR](https://github.com/FIWARE/dataModels/pull/377)
```
.....
  "allOf": [
    {
      "anyOf": [
        {
          "required": [
            "name"
          ]
        },
        {
          "required": [
            "shortName"
          ]
        }
      ]
    },
	{
      "required": [
        "id",
        "type",
        "routeType",
        "operatedBy"
      ]
    },
......
```

Reply:
Fixed in 3d7dacd